po/ja.po

```diff
@@ -10841,105 +11258,119 @@ msgid ""
 "        // to macOS (as opposed to iOS / wearOS / etc.) on Intel and "
 "PowerPC.\n"
 msgstr ""
+"// https://github.com/rust-lang/libc/issues/414、および  stat(2) の macOS 版"
+"のマニュアル ページの\n"
+"        // _DARWIN_FEATURE_64_BIT_INODE に関するセクションをご覧ください。\n"
+"        //\n"
+"        // 「これらのアップデートが利用可能になる前に存在していたプラット"
+"フォーム」とは、\n"
```

Comment:
I supose we want to leave the original sentence in parentheses too

Reply:
Can you suggest what's the change?

Reply:
I'll create a separate suggestion.